Bottom: fbe5c02ccf6099a9713fe1ce88cac01f66e9b39c
Top:    b8538c94bdd141fcd437e46be51ab9f5bf5c84f8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-09 19:57:16 +0000

Refresh of hid-common-provide-a-header-fi-0

---

diff --git a/src/hid/batch/batch.c b/src/hid/batch/batch.c
index 558f9fb..2c2b926 100644
--- a/src/hid/batch/batch.c
+++ b/src/hid/batch/batch.c
@@ -12,8 +12,6 @@
 #include "global.h"
 #include "hid.h"
 #include "data.h"
-#include "misc.h"
-#include "hid.h"
 
 #include "hid/common/actions.h"
 #include "hid/common/hidint.h"
